tests/db/test_group.py

```diff
@@ -79,6 +81,40 @@ async def test_group_db(engine: AsyncEngine) -> None:
                 spec_block_name="group",
             )
 
+        # test we can retrieve properties entered
+        entry = groups0[0]
+        check = await db.Group.get_row(session, entry.id)
+        assert check.db_id.id == entry.db_id.id
```

Comment:
Consider `assert check.db_id.id == entry.db_id.id, "Failed to retrieve group properties"`